### PR TITLE
bug 1490727: add new template for recurring payment thank you email 

### PR DESCRIPTION
**Pull request by joshJarr**
This PR adds a new email for the recurring payment. We're transcluding in the recurring payment info depending on if the email is generated from a recurring payment or not.

**In this PR** 
- https://trello.com/c/vSXHOpQh/13-as-a-user-i-want-to-receive-email-confirmation-that-my-payment-is-successful-so-that-i-can-be-sure-my-payment-has-worked

**Notes for QA**
- [ ] Ensure the one-time payment email is still sent and the links work
- [ ] Ensure recurring payment email is sent and links work
- [ ] Ensure recurring payment email reflect the same information as the thank you page.

**Screenshots** 
screenshot 2018-11-07 at 16 18 44

### CODE REVIEW

File: kuma/payments/jinja2/payments/email/thank_you/plain.ltxt [hunk 1]
```diff
@@ -9,3 +9,16 @@ We are committed to providing the best source of information about web standards
...
+{% if recurring_payment %}
+{% trans %}
+Managing your payments
+If you would like to manage your monthly payment, such as changing your card account details or the amount you pay, please contact {{ support_mail_link }}.
+Cancelling your payments
+If you would like to cancel your monthly payment, you are free to do so at any point. Please contact {{ support_mail_link }} to cancel.
+Payment Terms
+Please read our payment terms for more information.
+FAQ
+To find out more about why MDN is raising money through the one time and monthly payment programme, please visit our FAQ.
```
Comment (jwhitlock) [inline kuma/payments/jinja2/payments/email/thank_you/plain.ltxt]: The FAQ page should be added as a URL, and "programme" Americanized.

File: kuma/payments/jinja2/payments/email/thank_you/plain.ltxt [hunk 2]
```diff
@@ -9,3 +9,16 @@ We are committed to providing the best source of information about web standards
...
 {{_('The MDN Team')}}
+
+{% if recurring_payment %}
+{% trans %}
+Managing your payments
+If you would like to manage your monthly payment, such as changing your card account details or the amount you pay, please contact {{ support_mail_link }}.
+Cancelling your payments
+If you would like to cancel your monthly payment, you are free to do so at any point. Please contact {{ support_mail_link }} to cancel.
+Payment Terms
+Please read our payment terms for more information.
```
[the remark this answers is not in the document]
Comment (joshJarr) [reply kuma/payments/jinja2/payments/email/thank_you/plain.ltxt]: I've added 
```
{% trans %}
Please read our payment terms for more information:
{% endtrans %}
{{ url('payment_terms')|absolutify }}
```
Is this ok? Or is it preferred I render the payment_terms link in the context and pass this to the template?

File: kuma/payments/jinja2/payments/email/thank_you/plain.ltxt [hunk 3]
```diff
@@ -9,3 +9,16 @@ We are committed to providing the best source of information about web standards
 
 {{_('Thank you again,')}}
 {{_('The MDN Team')}}
+
+{% if recurring_payment %}
+{% trans %}
```
[the remark this answers is not in the document]
Comment (joshJarr) [reply kuma/payments/jinja2/payments/email/thank_you/plain.ltxt]: Good point! I've addressed this, thanks.

File: kuma/payments/jinja2/payments/email/thank_you/email.html [hunk 4]
```diff
@@ -115,6 +115,82 @@
...
+                                                  <tr style="border-top: 2px solid black;">
+                                                    <td width="640" align="left" style="background:#fff;">
+                                                      <table cellspacing="0" cellpadding="0">
+                                                        <tbody>
+                                                          <tr>
+                                                            <td align="left" style="font-family:Geneva,Tahoma,Verdana,sans-serif;font-size:14px;line-height:22px;color:#555555;padding-top:16px">
+                                                              <h3>{{_('FAQ')}}</h3>
+                                                              <p>
+                                                                {% trans faq_link=url('payments')|absolutify %}
+                                                                To find out more about why MDN is raising money through the one time and monthly payment programme, <a href="{{ faq_link }}">please visit our FAQ</a>.
```
[the remark this answers is not in the document]
Comment (joshJarr) [reply kuma/payments/jinja2/payments/email/thank_you/email.html]: 🇺🇸 Great spot!

File: kuma/payments/jinja2/payments/email/thank_you/plain.ltxt (hunk 1 above)
Comment (joshJarr) [reply kuma/payments/jinja2/payments/email/thank_you/plain.ltxt]: Updated, see above comments.

**Comment by joshJarr** [comment]
Thanks @jwhitlock, good spot with the "programme" spelling! I think copy is still under review, but I've updated this copy and formatting for now.


Plaintext copy reads:
```
Thank you joshJarr [josh.jarvis@potatolondon.com] for supporting MDN.

We are committed to providing the best source of information about web standards. Your support enables MDN to move even faster as we build more of the content and tools you use every day, and helps us make sure MDN can be available to everyone, everywhere.

Thank you again,
The MDN Team



Managing your payments
If you would like to manage your monthly payment, such as changing your card account details or the amount you pay, please contact mdn-support@mozilla.com.

Cancelling your payments
If you would like to cancel your monthly payment, you are free to do so at any point. Please contact mdn-support@mozilla.com to cancel.

Payment Terms
Please read our payment terms for more information:
https://localhost:8000/en-US/payments/terms

FAQ
To find out more about why MDN is raising money through the one time and monthly payment programs, please visit our FAQ.
https://localhost:8000/en-US/payments/
```

File: kuma/payments/tasks.py [hunk 5]
```diff
@@ -14,11 +14,14 @@
 
 @task
 @skip_in_maintenance_mode
-def contribute_thank_you_email(username, user_email):
+def payments_thank_you_email(username, user_email, recurring=False):
```
Comment (jwhitlock) [inline kuma/payments/tasks.py]: Thanks for updating the name!

File: kuma/payments/jinja2/payments/email/thank_you/plain.ltxt (hunk 2 above)
[the remark this answers is not in the document]
Comment (jwhitlock) [reply kuma/payments/jinja2/payments/email/thank_you/plain.ltxt]: This works well for the plain text email.

**Pull request merged**